docs/guides/installation.rst

```diff
@@ -1,64 +1,13 @@
 Installation
 ============
-Anvil-Extras is intended to be used as a dependency for other Anvil applications.
-
-1. Clone anvil-extras to your account:
-
-   .. image:: https://anvil.works/img/forum/copy-app.png
-      :height: 40px
-      :target: https://anvil.works/build#clone:C6ZZPAPN4YYF5NVJ=UGGCKFPRVZ7ELJH6RRZTHV6Y
-
-2. Add anvil-extras as a dependency to your own app(s):
+Anvil-Extras is intended to be used as a dependency for other Anvil applications:
 
    * From the gear icon at the top of your app's left hand sidebar, select 'Dependencies'
-   * From the 'Add a dependency' dropdown, select 'Anvil-Extras'
-
-That's it! You should now see the extra components available in your app's toolbox on the right hand side and all the other features are available for you to import.
-
-Upgrading
-=========
-To upgrade to a new release of Anvil-Extras, you will need to create a local copy of this repository and use git to update your copy of the app.
-
-Clone the Repository
---------------------
-* In your browser, navigate to your installed copy of the app within the Anvil IDE.
-* From the App Menu (with the gear icon), select 'Version History...' and click the 'Clone with Git' button.
-* Copy the displayed command to you clipboard.
-* In your terminal, navigate to a folder where you would like to create your local copy
-* Paste the command from your clipboard into your terminal and run it.
-* You should now have a new folder named 'anvil_extras'.
-
-Configure the Remote Repositories
----------------------------------
-Your local repository is now configured with a known remote repository pointing to your copy of the app at Anvil.
-That remote is currently named 'origin'. We will now rename it to something more meaningful and also add a second remote pointing to the repository on github.
-
-* In your terminal, navigate to your 'anvil_extras' folder.
-* Rename the 'origin' remote to 'anvil' with the command:
-
-.. code-block::
-
-    git remote rename origin anvil
-
-* Add the github repository with the command:
-
-.. code-block::
-
-    git remote add github git@github.com:anvilistas/anvil-extras.git
-
-Do the Upgrade
---------------
-To upgrade your app, we will now fetch the latest version from github to your local copy and push it from there to Anvil.
-
-* In your terminal, fetch the lastest code from github using the commands:
-
-.. code-block::
-
-    git fetch github
-    git reset --hard github/main
-
-* Finally, push those changes to your copy of the app at Anvil:
+   * In the buttons to the right of 'Add a dependency', click the 'Third Party' button
+   * Enter the id of the Anvil-Extras app: C6ZZPAPN4YYF5NVJ=UGGCKFPRVZ7ELJH6RRZTHV6Y
+   * Hit enter and ensure that the library appears in your list of dependencies
```

Comment:
ha yes hitting enter threw me for a second the first time I tried this!

Reply:
me too. That's there for my own benefit when I inevitably forget!